blockchain/chain_makers.go

```diff
@@ -238,6 +239,10 @@ func makeHeader(chain consensus.ChainReader, parent *types.Block, state *state.S
 		Number: new(big.Int).Add(parent.Number(), common.Big1),
 		Time:   time,
 	}
+	if chain.Config().IsKIP71ForkEnabled(header.Number) {
```

Comment:
I think the header verification logic should be updated so that  verifies the header's calculated `baseFee`

Reply:
You can see the verification code in `consensus/gxhash/consensus.go`

Reply:
@kjeom It's actually in [consensus/istanbul/backend/engine.go](https://github.com/klaytn/klaytn/blob/dev/consensus/istanbul/backend/engine.go#L176). Klaytn is using istanbul consensus engine.

Reply:
@blukat29 You are right, It needs to be fixed. Thanks

Reply:
It's moved to `consensus/istanbul/backend/engine.go`

Reply:
@kjeom, Checked. Thank you.
@blukat29, Good to know!  Thank you.